src/MOI_wrapper.jl

```diff
@@ -409,17 +388,18 @@ function MOI.optimize!(model::Model)
     return
 end
 
-# ==============================================================================
 # MOI accessors for the solution info.
 
+function mpb_solution_attribute(model::Model)
+    # We'd like to call MOI.get(model, MPBSolutionAttribute()), but it throws an
+    # error if not set rather than return `nothing`.
+    return get(model.modattr, MPBSolutionAttribute(), nothing)
```

Comment:
@blegat I was under the impression that this was going to change for `UniversalFallback`?

Reply:
Only for variable and constraint attributes